inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.cpp

```diff
@@ -244,4 +244,4 @@ void MKLDNNTensorIteratorNode::execute(mkldnn::stream strm) {
 bool MKLDNNTensorIteratorNode::created() const {
     return getType() == TensorIterator;
 }
-REG_MKLDNN_PRIM_FOR(MKLDNNTensorIteratorNode, TensorIterator);
+REG_MKLDNN_PRIM_FOR(MKLDNNTensorIteratorNode, MKLDNNPlugin::TensorIterator);
```

Comment:
Just wondering why do we need `MKLDNNPlugin::` here

Reply:
Without explicit namespace it clashes with TensorIterator from inference-engine/src/legacy_api/include/legacy/ie_layers.h.

Reply:
maybe it's better to remove global `using namespace InferenceEngine`?

Reply:
removed